src/observers/graph_observer.py

```diff
@@ -98,22 +98,25 @@ def _extract_features(self, agent) -> Dict[str, float]:
     res["vel"] = state[int(StateDefinition.VEL_POSITION)]
 
     # get information related to goal
-    goal_center = agent.goal_definition.goal_shape.center[0:2]
+    goal_center = agent.goal_definition.goal_shape.center[0:2] #Velocity the agent should have at goal
+    goal_velocity = np.mean(agent.goal_definition.velocity_range)    
+    
     goal_dx = goal_center[0] - res["x"] # distance to goal in x coord
     goal_dy = goal_center[1] - res["y"] # distance to goal in y coord
     goal_theta = np.arctan2(goal_dy, goal_dx) # theta for straight line to goal
     goal_d = np.sqrt(goal_dx**2 + goal_dy**2) # distance to goal
+    res["goal_vel"] = goal_velocity # Could also include delta between cur_velocity and goal_velocity
     res["goal_d"] = goal_d
     res["goal_theta"] = goal_theta
 
-    if self._normalization_enabled:
+    """if self._normalization_enabled:
```

Comment:
Das sollten wir wieder aktivieren.
```suggestion
    if self._normalization_enabled:
```